synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -207,6 +213,13 @@ MainWindow::init_menus()
 		sigc::ptr_fun(studio::App::dialog_help)
 	);
 
+#if GTK_CHECK_VERSION(3, 20, 0)
```

Comment:
there are other places with that other macro checking :)

Reply:
Do you mean I should use `GTK_CHECK_VERSION(3, 20, 0)` and not `GTKMM_MINOR_VERSION >= 20`?

Or are there more I missed in this file?

Reply:
Hope you didn't mean this changes I missed https://github.com/synfig/synfig/pull/1776/commits/d5768c69876863ad48f1653b25c40cd1fad5c49e